### DISCUSSION TITLE
[Question][Hw1] Grading of cracks

### DISCUSSION THREAD

**Discussion by KalasLavas** [prosyslab-classroom/cs348-information-security#668] [Q&A]
Name: Aziz Huseynov

How are cracks graded? I improved my code for substitute crack, it has higher `sim` score, but works a bit slower. But on gradescope it gets lower score, is runtime and memory usage considered in the grading? Or is it only output of `sim`?

**Comment by KalasLavas** [comment]
Seems like at least runtime is involved, I optimized the code (without changing how it work) and score increased

**Comment by yeonhee-ryou** [comment]
Yes GradeScope limits the total execution time
